app/views/content_items/specialist_document.html.erb

```diff
@@ -1,28 +1,38 @@
 <% content_for :simple_header, true %>
 
 <%= render 'shared/title_and_translations', content_item: @content_item %>
-<%= render 'shared/metadata', content_item: @content_item %>
-<%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
 
-<div class="grid-row sidebar-with-body">
-  <% if @content_item.contents.any? %>
-    <div class="column-third">
-      <%= render 'components/contents-list', contents: @content_item.contents %>
+  <div class="grid-row">
+    <div class="column-two-thirds" data-module="sticky-element-container">
+      <%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
+      <%= render 'components/publisher-metadata', @content_item.content_metadata %>
+      <%= render 'components/important-metadata', items: @content_item.metadata[:other] %>
+      <% if @content_item.contents.any? %>
+        <div id="contents-list">
+          <%= render 'components/contents-list', contents: @content_item.contents %>
+        </div>
+      <% end %>
+      <%= render 'govuk_component/govspeak', @content_item.govspeak_body %>
+      <% if @content_item.continuation_link %>
+        <%= render(
+            'govuk_component/button',
+            start: true,
+            href: @content_item.continuation_link,
+            text: "Find out more",
+            info_text: @content_item.will_continue_on
+        ) %>
+      <% end %>
+      <div class="responsive-bottom-margin">
+        <%= render 'components/published-dates', {
+              published: @content_item.published,
+              last_updated: @content_item.updated,
+              history: @content_item.history
+            } %>
+      </div>
+      <div class="contents-list-link-wrapper" data-sticky-element>
```

Comment:
I feel like the styling applied to `contents-list-link-wrapper` means that it should be part of the back-to-top component.
Then we could just wrap the component in a div with the `data-sticky-element` on for the sticky behaviour?

Reply:
I considered this but decided against it because it ties that particular use case into the component and makes it less flexible - at the moment you can use a back to top wherever you like and decorate it at will and I think that's worth preserving. I considered creating a second component that calls the existing one but it seems like a lot of overhead for essentially a single div wrapper. Also, I would imagine that a lot of the current templates will be collapsed into fewer - the 4 types that have contents lists may well end up in a single body partial, so we'd be looking at a component that only got used once. Happy to take further opinions on this though. @andysellick @nickcolley what are your thoughts?